### PR TITLE
MERPWH-203 Create unit tests for module merp_inventory

### PR DESCRIPTION
**Pull request by denisvalenchyts**

### CODE REVIEW

File: merp_custom_logotype/tests/__init__.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
```
Comment (nedaszilinskas) [inline merp_custom_logotype/tests/__init__.py:1]: This is no longer needed. You can use [pylint-odoo](https://github.com/OCA/pylint-odoo) tool to validate code.

Please use creds as in other files:
Copyright 2019 VentorTech OU
Part of Ventor modules. See LICENSE file for full copyright and licensing details.

File: merp_custom_logotype/tests/__init__.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+
+from . import test_checking_logotype
```
Comment (nedaszilinskas) [inline merp_custom_logotype/tests/__init__.py:3]: why is this here? I think there was another MR for this

File: merp_inventory/tests/test_check_default_location.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        self.product = self.env['product.template'].create({
+            'name': 'new_product'
+        })
+
+    def test_check_stock_inventory_location(self):
+        self.user.write({
+            'company_id': self.company.id,
+            'company_ids': [(4, self.company.id, 0)]
+        })
+        product = self.env['product.template'].sudo(self.user.id).browse(self.product.id)
```
Comment (nedaszilinskas) [inline merp_inventory/tests/test_check_default_location.py]: I think this line could just be `product = self.product.sudo(self.user.id)`, no need to re-browse.

**Comment by nedaszilinskas** [comment]
Could write simple tests on `finish_inventory` and `return_inventory` methods to have coverage on these as well.

**Comment by denisvalenchyts** [comment]
This test checks what inventory default location value will be set.

File: merp_custom_logotype/tests/__init__.py (hunk 2 above)
Comment (denisvalenchyts) [reply merp_custom_logotype/tests/__init__.py:3]: I didn't switch to branch 12.0 when created this branch
